dask/dataframe/hashing.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+
+from pandas.core.categorical import is_categorical_dtype
+
+
+def hash_pandas_object(obj):
```

Comment:
I'll hold off for now and let you or wait on after we figure out if the API should change and to see if this needs to tweak in light of what is going into pandas.